imagersite/imager_images/models.py

```diff
@@ -53,3 +53,19 @@ class Album(models.Model):
 
     def __str__(self):
         return self.title
+
+
+@python_2_unicode_compatible
+class Face(models.Model):
+    photo = models.ForeignKey(Photo,
+                              related_name='faces',
+                              null=False,
+                              )
```

Comment:
I'd much rather see this indented like so:

```python
    photo = models.ForeignKey(
        Photo,
        related_name='faces',
        null=False,
    )
```
It's a lot easier to read since the left margins are closer together.  I think you'll find most Django coders follow this approach.